app/main.py

```diff
@@ -15,13 +15,13 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-is_mutable = {"mutable": [], "immutable": []}
+sorted_variables = {"mutable": [], "immutable": []}
 variables = [lucky_number, pi, one_is_a_prime_number, name,
              my_favourite_films, profile_info, marks, collection_of_coins]
 for i in range(len(variables)):
     if (isinstance(variables[i], set)
```

Comment:
Shouldhave make it in a tuple like: 
if isinstance(variable[i], (int, float, str, bool, tuple)):